src/components/capture-area/CaptureAreaSearchValue.vue

```diff
@@ -1,148 +1,155 @@
-          ></v-text-field
-        ></v-col>
-        <v-col
-          ><v-btn
-            color="red"
-            icon="mdi-delete"
-            rounded="lg"
-            variant="tonal"
-          ></v-btn
-        ></v-col>
-      </v-row>
+        <v-card-text>
+            <v-row class="mb-4" no-gutter>
+                <v-col
+                    ><v-text-field
+                        label="Regular Expression"
+                        placeholder="Enter a regular expression"
+                        variant="outlined"
+                        dense
+                        clear-icon="mdi-close-circle"
+                        :rules="[
+                            (v) => !!v || 'Regular expression is required',
+                        ]"
+                    ></v-text-field
+                ></v-col>
+            </v-row>
+
+            <v-row class="mb-4" no-gutters cols="12">
+                <v-col cols="2">is</v-col>
+                <v-col cols="3"
+                    ><v-combobox
+                        class="pr-2 pl-2"
+                        label="Position"
+                        :items="['After', 'Before', 'Between']"
+                        variant="underlined"
+                    ></v-combobox
+                ></v-col>
+                <v-col cols="7"
+                    ><v-text-field
+                        cols="1"
+                        label="Regular Expression"
+                        placeholder="Enter a regular expression"
+                        variant="outlined"
+                        dense
+                        clear-icon="mdi-close-circle"
+                        :rules="[
+                            (v) => !!v || 'Regular expression is required',
+                        ]"
+                    ></v-text-field
+                ></v-col>
+            </v-row>
 
-      <!-- TODO: this can be repeated many times -->
-      <v-row>
-        <v-col
-          ><v-combobox
-            clearable
-            label="Logical Operator"
-            :items="['AND', 'OR', 'NOT']"
-            variant="underlined"
-          ></v-combobox
-        ></v-col>
-        <v-col
-          ><v-combobox
-            clearable
-            label="Position"
-            :items="['After', 'Before', 'Between']"
-            variant="underlined"
-          ></v-combobox
-        ></v-col>
-        <v-col
-          ><v-text-field
-            label="Regular Expression"
-            placeholder="Enter a regular expression"
-            variant="outlined"
-            dense
-            clearable
-            clear-icon="mdi-close-circle"
-            :rules="[(v) => !!v || 'Regular expression is required']"
-          ></v-text-field
-        ></v-col>
-        <v-col
+            <v-row class="mb-4" no-gutters>
+                <v-col cols="2"
+                    ><v-combobox
+                        label="Logic"
+                        :items="['AND', 'OR', 'NOT']"
```

Comment:
Still tbd if we can actually implement this in MVP.

Reply:
Yes can be easily removed if not needed. This component has to be updated nonetheless in the upcoming issues.